src/Test/TestCases.Xaml/TestCases.Xaml.csproj

```diff
@@ -0,0 +1,30 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp3.0</TargetFrameworks>
```

Comment:
This should also target netcoreapp2.1 to check the portable System.Xaml.

Reply:
Done